docs/using-the-badge/replacing-parts.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Replace badge components
+weight: 5
+---
+
+If you need to replace your badge components, you can order a replacement part from ebay, aliexpress etc.
+
+## Replace your screen
+
+The badge uses a GC9A01 1.28" round LCD module.
+
+Some links to the correct screen:
+
+- [Aliexpress with connector soldered on](https://www.aliexpress.com/item/1005004392903184.html)
+- [Aliexpress with loose connector](https://www.aliexpress.com/item/1005005936868343.html)
+- [Amazon](https://www.amazon.co.uk/ARCELI-Arduino-Resolution-Interface-Display/dp/B0CFXVD9HX/)
+
+Carefully pull the connector of the display up at a 90 degree angle. If you bought a new display with a connector that is soldered on, carefully push the connector of the new display into the slots. The display is held in place by friction, so this is all you need to do.
+
+If you bought a display with a loose connector you have to also push the display onto the connector. If you have soldering tools, you can permanently affix the display to the connector.
```

Comment:
If they've bought a display without the pin header soldered, they'll have to solder it on the screen side, it won't make electrical contact if they just push it in.

Reply:
oh that sucks